Gemfile.lock

```diff
@@ -124,6 +124,3 @@ PLATFORMS
 
 DEPENDENCIES
   github-pages
-
-BUNDLED WITH
-   1.10.6
```

Comment:
@philsturgeon my old bundler removed that. after upgrading to 1.11.2 and running bundle install again it didn't add that back, either.